src/utils/stringUtils.ts

```diff
@@ -30,7 +37,7 @@ export class stringUtils {
      * }
     */
 
-    public static URLParse(string: string) {
+    public static urlParse(string: string) {
```

Comment:
You missed it again?? CHANGE `string` to `url` my god. Parameter names need to be concise.